medusa/image_cache.py

```diff
@@ -163,6 +163,8 @@ def replace_images(series_obj):
     """
     remove_images(series_obj)
     fill_cache(series_obj)
+    # Update the increment, to bypass the client side caching.
+    app.INC += 1
```

Comment:
Maybe this feels a bit awkward. But we and hide this in a simple decorator. Then a function that should up the incremental, would look like:
```
@invalidate_browser_cache
def replace_images(series_obj):
    """
    Replace cached images for a series based on image type.

    :param series_obj: Series object
    """
    remove_images(series_obj)
    fill_cache(series_obj)
```